src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -355,6 +356,10 @@ public void cannotCheckOutWhenLoaneeCannotBeFound() {
     assertThat(response.getJson(), hasErrorWith(allOf(
       hasMessage("Could not find user with matching barcode"),
       hasUserBarcodeParameter(steve))));
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("Item is not loanable"),
+      hasItemBarcodeParameter(smallAngryPlanet))));
```

Comment:
How can it be determined that an item is not loanable when the borrower cannot be found?

As I understand it, the decision as to whether an item is loanable is based upon the loan policy, which is based upon the circulation rules and those require a patron group.

Reply:
I agree, there is no point in checking whether the item is not loanable or item limit is reached without a loan policy, so I've disabled those validations for such case and also removed this piece from the test.